### DISCUSSION TITLE
Docker image Ubuntu version

### DISCUSSION THREAD

**Discussion by Hoyyyaard** [StanfordVL/BEHAVIOR-1K#995] [Q&A] [answered]
Can you please provide the omnigibson docker in ubuntu20.04 for the version 1.0.0? As current docker image is build on ubuntu22.04. Thanks!

**Comment by cgokmen** [comment, marked as answer]
Hi, we build our Docker images on top of the official Isaac Sim image and thus are not able to change the Ubuntu version included.
